examples/compute_structure.shader_test

```diff
@@ -0,0 +1,151 @@
+# // Source code in OpenCL C:
```

Comment:
VkRunner supports a section called [comment] which it ignores. It might be nicer to put the original source code under that so that we don’t have to add # to the beginning of every line.

Reply:
Done.